tests/base/data_fin001/fromInteger.idr

```diff
@@ -0,0 +1,22 @@
+
+-- this is to test that changes introduced for #2032 do not ruin other things
+-- (#2032 itself is a performance issue, I don't know how to properly test for that)
```

Comment:
There's a `benchmarks` folder with some benchmarks in it, but I don't know the history of that stuff and I can't seem to run the benchmarks locally using the `bench.sh` script (might be because I am on a mac). If that is not being maintained, I don't think there is any other way for you to create tests beyond these which ensure functionality.